### ISSUE TITLE
Is this a bug in the loss function?

### ISSUE THREAD

**Issue by hojjatabdollahi** [aloyschen/tensorflow-yolo3#17]
In `yolo3_model.py`  in the function `yolo_loss` at the end you are dividing by `yolo_output` shape:
    `class_loss = tf.reduce_sum(class_loss) / tf.cast(tf.shape(yolo_output[0])[0], tf.float32)`
Shouldn't it be `yolo_output[index]` instead of `yolo_output[0]`? You are using it for all the losses.

**Comment by zkk0911** [comment]
yolo_output[0] refers to batch size
